cart/domain/cart/cart.go

```diff
@@ -750,12 +801,54 @@ func (p PricedItems) TotalItems() map[string]domain.Price {
 	return p.totalItems
 }
 
-//ShippingItems - returns the Price per ShippingItems - map key is deliverycode
+// GetAllTotalItems – return all total items
+func (p PricedItems) GetAllTotalItems() []PricedTotalItem {
```

Comment:
Duplicate with function TotalItems

we should decide on one of the options

Reply:
Party true, TotalItems return map[string] of Price which unfortunately can't be used in graphql. I'll move this one into interface layer.